test/jquery.ime.test.fixtures.js

```diff
@@ -2986,5 +2986,13 @@ var testFixtures = [
 			{ input: 'Ẑh', output: 'Zh', description: 'Venetian Zh' }
 		],
 		inputmethod: 'vec-GVU'
+	},
+	{
+		description: 'Telugu Transliteration test',
+		tests: [
+			{ input: 'j~nu', output: 'జ్ఞు', description: Telugu j~nu -> జ్ఞు' },
+			{ input: 'j~nE', output: 'జ్ఞే', description: Telugu j~nE -> జ్ఞే' },
```

Comment:
Please remove trailing comma here. Also missing quote for description
Also please test once to see all tests pass.